pkgs/desktops/plasma-5/kinfocenter/default.nix

```diff
@@ -0,0 +1,104 @@
+{ mkDerivation
+, lib
+, extra-cmake-modules
+, kdoctools
+, qtbase
+, qttools
+, kcmutils
+, kcompletion
+, kconfig
+, kconfigwidgets
+, kcoreaddons
+, kdbusaddons
+, kdeclarative
+, ki18n
+, kiconthemes
+, kio
+, kirigami2
+, kpackage
+, kservice
+, kwayland
+, kwidgetsaddons
+, kxmlgui
+, solid
+, systemsettings
+, dmidecode
+, fwupd
+, libraw1394
+, libusb
```

Comment:
```
error: Function called without required argument "libusb" at /home/shawn/.cache/nixpkgs-review/pr-212921/nixpkgs/pkgs/desktops/plasma-5/kinfocenter/default.nix:28, did you mean "librsb", "libusb1" or "ibus"?
```

I am receiving this error when building with nixkpgs-review. I assume that its should be `libusb1`? if i change it this way it at least builds fine

Reply:
You're right that it should be `libusb1`, but there is an alias defined for `libusb` so that should work too (I'm running it here. Maybe nixpkgs-review doesn't load the aliases?